packages/context/test/unit/context.unit.ts

```diff
@@ -272,6 +272,24 @@ describe('Context', () => {
     });
   });
 
+  describe('findByTagValue', () => {
+    it('returns matching binding', () => {
+      const b1 = ctx.bind('foo').tagValue('t1', 'val1');
+      ctx.bind('bar').tag('t2');
+      ctx.bind('baz').tagValue('t3', 1);
```

Comment:
Would you mind using a more descriptive strings than bar/baz/t1/t2?

For example:
 - foo => ProductController
 - bar => ProductRepository
 - baz => db
 - t1 => controller
 - t2 => repository
 - t3 => datasource